packages/ses/src/get-anonymous-intrinsics.js

```diff
@@ -26,10 +38,7 @@ function makeArguments() {
  * @returns {Object}
  */
 export const getAnonymousIntrinsics = () => {
-  const InertFunction = Function.prototype.constructor;
-
-  const SymbolIterator = (typeof Symbol && Symbol.iterator) || '@@iterator';
-  const SymbolMatchAll = (typeof Symbol && Symbol.matchAll) || '@@matchAll';
```

Comment:
Drive-by fix. These lines were defective, not testing the value of each typeof expression. Beyond removing the defective code, using `iteratorSymbol` and `matchAllSymbol` from `commons.js` is also spiritually different than the original intended semantics. As a matter of fact, we never supported versions before `Symbol` because of this defect. As a matter of stated supported versions, we no longer intend to.

Fixes #828